server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevelStartup.java

```diff
@@ -58,6 +62,11 @@ protected void configureLevel() {
       RegisterRules.class);
     add(BuiltInQProfileLoader.class);
     addIfStartupLeader(
+      // TODO Should we put it in level 2 ?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)